release/gcb/docker_tag_push_lib.sh

```diff
@@ -53,14 +53,16 @@ function add_extra_artifacts_to_tar_images() {
   fi
 
   for TAR_PATH in "${OUT_PATH}"/docker/*.tar.gz; do
-    BASE_NAME=$(basename "$TAR_PATH")
-    TAR_NAME="${BASE_NAME%.*}"
-    IMAGE_NAME="${TAR_NAME%.*}"
-
     # if no docker/ directory or directory has no tar files
-    if [[ "${IMAGE_NAME}" == "*" ]]; then
+    if [[ "${TAR_PATH}" == *"/docker/*.tar.gz" ]]; then
```

Comment:
Does this really work? 

Why not:

```
if [[ "${TAR_PATH}" == "${OUT_PATH}/docker/*.tar.gz" ]]
```

Reply:
It does, but using `${OUT_PATH}` makes more sense.

Reply:
I see, thanks.